### ISSUE TITLE
Make sure that I am not getting extraneous gradient signal

### ISSUE THREAD

**Issue by scottviteri** [scottviteri/MarkovianTraining#88]
Put generate in a torch.no_grad().
Actually just put the whole rao trajectory generation into torch.no_grad().
